src/Sulu/Bundle/AdminBundle/Resources/js/components/containerSizeAware/__mocks__/containerSizeAware.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+
+export default function containerSizeAware(Component: ReactClass<*>) {
+    class ContainerSizeAwareComponent extends React.Component {
+        component: *;
+
+        componentDidMount() {
+            this.component.containerDidMount();
+            if (this.props.mountSpy) {
+                Promise.resolve().then(this.props.mountSpy);
+            }
+        }
+
+        setComponent = (c: *) => this.component = c;
+
+        render() {
+            const props = {
+                ...this.props,
+                containerWidth: 640,
+                containerHeight: 360,
+                ref: this.setComponent,
+            };
+
+            return <Component {...props} />;
+        }
+    }
+
+    return ContainerSizeAwareComponent;
```

Comment:
This is just a mock for the tests. In the real component the `displayName` is set